authentication-tokens/cli_examples.sh

```diff
@@ -0,0 +1,29 @@
+# CLI and CloudShell examples for generating Aurora DSQL authentication tokens
```

Comment:
The structure of the `aws-samples/aurora-dsql-samples` repo is currently broken out by language. Would these samples be better served under the language-specific sub folders?

If I was looking for Java samples I would likely open the `java` directory to look for an authentication token sample for Java.